src/Symfony/Component/DependencyInjection/Argument/ServiceLocatorArgument.php

```diff
@@ -21,7 +23,7 @@
  *
  * @experimental in version 3.3
  */
-class ServiceLocatorArgument implements ArgumentInterface
+class ServiceLocatorArgument extends Definition implements ArgumentInterface
```

Comment:
won't this cause issues with recursive passes processing Definition objects ? Is it really a Definition ?